api/build.go

```diff
@@ -362,6 +362,106 @@ func GetBuilds(c *gin.Context) {
 	c.JSON(http.StatusOK, b)
 }
 
+// swagger:operation GET /api/v1/repos/{org} builds GetOrgBuilds
+//
+// Create a build in the configured backend
+//
+// ---
+// x-success_http_code: '200'
+// produces:
+// - application/json
+// parameters:
+// - in: path
+//   name: org
+//   description: Name of the org
+//   required: true
+//   type: string
+// - in: header
+//   name: Authorization
+//   description: Vela bearer token
+//   required: true
+//   type: string
+// responses:
+//   '200':
+//     description: Successfully retrieved the build
```

Comment:
This description doesn't sound correct. This is getting a list of builds, right?

Reply:
Fixed.